main.py

```diff
@@ -0,0 +1,61 @@
+import requests
+import time
+import os
+
+from dotenv import load_dotenv
+
+load_dotenv()
+
+API_KEY = os.getenv('API_KEY') #IEX CLOUD API
+
+WEBHOOKS_KEY = os.getenv('WEBHOOKS_KEY') #WEBHOOKS KEY
+
+
+
+IFTTT_URL = f"https://maker.ifttt.com/trigger/notify/with/key/{WEBHOOKS_KEY}" #IFTTT URL
+
+
+
+PRICE_DROP_THRESHOLD = 0.25 #THRESHOLD FOR PRICE DROP
```

Comment:
Very good use of variables to avoid magic numbers- falls within good coding standards